src/main/java/io/vertx/core/json/jackson/JsonArrayDeserializer.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2011-2023 Contributors to the Eclipse Foundation
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ */
+package io.vertx.core.json.jackson;
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.DeserializationContext;
+import com.fasterxml.jackson.databind.JsonDeserializer;
+import io.vertx.core.json.JsonArray;
+
+import java.io.IOException;
+import java.util.List;
+
+class JsonArrayDeserializer extends JsonDeserializer<JsonArray> {
```

Comment:
It seems the source is not formatted correctly. Can you make sure your IDE takes the `https://github.com/eclipse-vertx/vert.x/blob/master/.editorconfig` file into account?